### DISCUSSION TITLE
Is it possible to list a root level page or leaf-bundle in "Navigation"?

### DISCUSSION THREAD

**Discussion by n-arakawa** [thegeeklab/hugo-geekdoc#562] [Q&A] [answered]
It is allowed to create leaf bundles directly under the "content" directory.

https://gohugo.io/content-management/page-bundles/#examples-of-leaf-bundle-organization

I tried following content structure.

```
content/
├── Section1
│   ├── _index.md
│   └── page.md
├── leaf_bundle
│   └── index.md
└── page1.md
```

The navigation only shows "Section1".

 Is it possible to list a root level page or leaf-bundle in "Navigation"?

**Comment by xoxys** [comment, marked as answer]
The content structure depends on the menu you want to use. Using the auto-generated [file-tree menu](https://geekdocs.de/usage/menus/#file-tree-menu) only branch bundles are supported. If you want to use a mix of multiple types in your content structure, you would need to switch to the manual [bundle menu](https://geekdocs.de/usage/menus/#bundle-menu).

**Reply by n-arakawa** [reply]
Thanks.
But I prefer the "auto-generated" method!

**Comment by n-arakawa** [comment]
Following customization works to some extent.
//  Not checking for bad side effects etc.


Replacing `Sections` with  `Pages` in `hugo-geekdoc/layouts/partials/menu-filetree.html`  
```
{{ template "tree-nav" dict "sect" .Site.Home.Sections "current" $current }}
=>
{{ template "tree-nav" dict "sect" .Site.Home.Pages "current" $current }}
```

**Reply by xoxys** [reply]
Sure, it's up to you to customize partials to your needs. The intention of the file tree menu is to work with branch bundles.
